src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -91,9 +91,6 @@ protected void doStart() throws ElasticsearchException {
         adapter.txMetric.clear();
         transport.transportServiceAdapter(adapter);
         transport.start();
-        if (transport.boundAddress() != null && logger.isInfoEnabled()) {
-            logger.info("{}", transport.boundAddress());
```

Comment:
just logging a tostring representation of the bound address seemed totally useless to me, you can get infos for that when the NettyTransport starts anyway...